.github/workflows/test.yml

```
@@ -53,11 +36,36 @@ jobs:
         python -m pip install --upgrade pip
         pip install tox tox-gh-actions
 
-    - name: Run test
+    - name: Run tests with PyTest 4
       run: tox
+      if: ${{ matrix.python-version != '3.10' }}
       env:
         PLATFORM: ${{ matrix.os }}
-        PYTEST_MAJOR_VERSION: ${{ matrix.pytest-version }}
+        PYTEST_MAJOR_VERSION: 4
+        PYTEST_PLUGINS: pytest_github_actions_annotate_failures
+
+    - name: Run tests with PyTest 5
+      run: tox
+      if: ${{ matrix.python-version != '3.10' }}
```

Pytest >= 5 doesn't support Python 2.7, so like this? (Though `tox` command simply exits without testing with Python 2.7 and pytest 5)
```suggestion
      if: ${{ matrix.python-version != '2.7' && matrix.python-version != '3.10' }}
```

---

Wow, that's weird on the 2.7 pytest 5 step:

```
Run tox
  tox
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/2.7.18/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/2.7.18/x64/lib/pkgconfig
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/2.7.18/x64/lib
    PLATFORM: ubuntu-lates
    PYTEST_MAJOR_VERSION: 5
    PYTEST_PLUGINS: pytest_github_actions_annotate_failures
___________________________________ summary ____________________________________
  congratulations :)
```

It does nothing and passes.